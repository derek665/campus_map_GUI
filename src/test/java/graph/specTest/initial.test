# Tests that a graph can be created and printed

# Create a graph
CreateGraph graph1

# Print nodes in graph
ListNodes graph1