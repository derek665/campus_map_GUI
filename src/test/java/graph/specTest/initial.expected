# Tests that a graph can be created and printed

# Create a graph
created graph1

# Print nodes in graph
graph1 contains: